onnxruntime/test/providers/cpu/nn/pool_op_test.cc

```diff
@@ -1264,6 +1264,25 @@ TEST(PoolTest, LpPool) {
   test.Run();
 }
 
+TEST(PoolTest, LpPoolWithNegativeNumbers) {
+  OpTester test("LpPool");
+
+  test.AddAttribute("p", static_cast<int64_t>(1));
+  test.AddAttribute("auto_pad", "");
+  test.AddAttribute("strides", std::vector<int64_t>{2});
+  test.AddAttribute("pads", vector<int64_t>{0, 0, 0});
+  test.AddAttribute("kernel_shape", vector<int64_t>{2});
+
+  std::vector<float> x_vals = {0.2, -0.6};
```

Comment:
On Windows, it doesn't compile because by default the type is double and it warns on truncation - can you please add 0.2f, and so on ? The same for `expected_vals ` as well.

Reply:
#[warning]onnxruntime\test\providers\cpu\nn\pool_op_test.cc(1276,41): Warning C4305: 'initializing': truncation from 'double' to '_Ty'
    73>C:\a\1\s\onnxruntime\test\providers\cpu\nn\pool_op_test.cc(1276,41): warning C4305: 'initializing': truncation from 'double' to '_Ty' [C:\a\1\b\RelWithDebInfo\onnxruntime_test_all.vcxproj]
                 with
                 [
                     _Ty=float
                 ]
##[warning]onnxruntime\test\providers\cpu\nn\pool_op_test.cc(1279,42): Warning C4305: 'initializing': truncation from 'double' to '_Ty'
    73>C:\a\1\s\onnxruntime\test\providers\cpu\nn\pool_op_test.cc(1279,42): warning C4305: 'initializing': truncation from 'double' to '_Ty' [C:\a\1\b\RelWithDebInfo\onnxruntime_test_all.vcxproj]
                 with
                 [
                     _Ty=float
                 ]

Reply:
done.